lib/lspserver/include/lspserver/DraftStore.h

```diff
@@ -0,0 +1,59 @@
+//===--- DraftStore.h - File contents container -----------------*- C++ -*-===//
```

Comment:
If we copy & paste this from LLVM monorepo, we should add a link so everyone knows it when they see the code. In addition, it makes it easier to sync with the upstream if anything changes.

Reply:
We can provide a link here: https://github.com/inclyc/nixd/blob/main/lib/lspserver/README.md

However, using git submodule & sync with upstream is not applicable, because clangd has it own logic about how to index & parsing files, coupled with the `DraftStore`, `SourceCode`, etc.